src/compas_slicer/post_processing/generate_raft.py

```diff
@@ -0,0 +1,178 @@
+import logging
+import math
+
+import compas_slicer
+from compas_slicer.geometry import Layer
+from compas_slicer.geometry import Path
+
+from compas.geometry import Point
+from compas.geometry import Line
+from compas.geometry import Vector
+from compas.geometry import bounding_box_xy
+from compas.geometry import offset_polygon
+from compas.geometry import intersection_line_line
+from compas.geometry import offset_line
+
+logger = logging.getLogger('logger')
+
+__all__ = ['generate_raft']
+
+
+def generate_raft(slicer,
+                  raft_offset=10,
+                  distance_between_paths=10,
+                  direction="horizontal",
```

Comment:
I wonder if it would be better to change the directions naming from vertical, horizontal, diagonal, to something like x_axis, y_axis and xy_axis? Because like that it can be a bit confusing what is horizontal and vertical

Reply:
the `xy_axis `alone would also be confusing. Maybe this could be something like `xy_axis_diagonal`?

Reply:
hmm yea actually I had called it x_axis and y_axis in the beginning, but then I added "diagonal" and then I thought it makes sense to have all of them sound similar. something like xy_axis_diagonal could work I guess